source/code-snippets/typescript/dot-notation.ts

```diff
@@ -0,0 +1,31 @@
+// start-no-error
+interface TestType {
+  field: {nested : number}; 
+}
+const database = client.db("<your db>");
+const collection = database.collection<TestType>("<your collection>");
+await collection.updateOne(
+  {},
+  {$set: { "field.nested" : "A string"}}
```

Comment:
it would be good to use consistent spacing for curly braces, e.g. `{ key: value }`

Reply:
Ran through formatter!